src/cpp/core/file_lock/LinkBasedFileLock.cpp

```diff
@@ -432,13 +441,16 @@ Error LinkBasedFileLock::acquire(const FilePath& lockFilePath)
 
    // write the lock file -- this step _must_ be atomic and so only one
    // competing process should be able to succeed here
-   error = writeLockFile(lockFilePath);
+   Error writeError = writeLockFile(lockFilePath);
    if (error)
```

Comment:
Seems like there's no way to enter this block after renaming the variable?

Reply:
Doh; thanks for catching that. This should've also been renamed `writeError`.